eclipse-che/che-theia
# chore: Store user preferences in a config-map

Signed-off-by: Igor Vinokur <ivinokur@redhat.com>



### What does this PR do?
Use config-map to store and restore user preferences. This gives an ability not to use che-server API for storing preferences.

### Screenshot/screencast of this PR



### What issues does this PR fix or reference?

https://github.com/eclipse/che/issues/20622

### How to test this PR?

1. Open a workspace
2. Change some user preferences, stop the workspace.
3. Start a new workspace.
See: settings are present in the new workspace.

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)



### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### Happy Path Channel

HAPPY_PATH_CHANNEL=stable
